datamodel/BUILD

```diff
@@ -173,34 +173,37 @@ cc_library(
             ],
         ),
     }),
-    copts = [
-        "-Iexternal/nlohmann/include",
-    ],
     defines = select({
         "//meta_package:rdmn_vk_rasterizer_backend": ["RDMN_VULKAN"],
         "//meta_package:rdmn_gl_rasterizer_backend": ["RDMN_OPENGL"],
     }),
     include_prefix = "rcbe-engine",
     includes = ["include"],
     linkopts = select({
-        "//meta_package:rdmn_vk_rasterizer_backend": ["-lvulkan"],
-        "//meta_package:rdmn_gl_rasterizer_backend": ["-lGL"],
-    }) + [
-        "-lpthread",
-    ],
+        "//meta_package:rdmn_vulkan_linux": ["-lvulkan"],
+        "//meta_package:rdmn_gl_linux": ["-lGL"],
+        "//conditions:default": [],
+    }) + select({
+        "@bazel_tools//src/conditions:linux": ["-lpthread"],
+        "//conditions:default": [],
+    }),
     strip_include_prefix = "include",
     visibility = ["//visibility:public"],
     deps = select({
         "//meta_package:rdmn_gl_rasterizer_backend": ["//gl_extensions"],
         "//meta_package:rdmn_vk_rasterizer_backend": [],
+    }) + select({
+        "//meta_package:rdmn_vulkan_windows": ["@vulkan_windows//:vulkan_cc_library",],
```

Comment:
Maybe we should use this for linux as well, instead of just linking to `-lvulkan`?

Reply:
This approach basically does the same thing for locally installed Vulkan. so I see no reason to do it.

Reply:
Ok. leave like that for a while, we'll return to it later.